scripts/_regen_receipt_merge.py

```
@@ -0,0 +1,68 @@
+from pathlib import Path
+
+ROOT = Path(__file__).resolve().parent.parent
+RECEIPT_PATH = ROOT / "crates/labcolors-core/contracts/clean-set-srgb8-v1/receipt-v1.json"
+PIN_PATH = ROOT / "crates/labcolors-core/contracts/clean-set-srgb8-v1/receipt-v1.sha256"
+
+
+def canonical_json(value):
+    return json.dumps(value, allow_nan=False, ensure_ascii=True, indent=2, sort_keys=True) + "\n"
+
+
+def git_show(path):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `git_show`

(ANN202)